lib/tasks.js

```diff
@@ -587,4 +587,58 @@ module.exports = function(self, options) {
     await resolveDeferredRelationships();
     console.log('... Done.');
   };
+
+  self.enableCleanPagesTreeTask = function() {
+    self.addTask('clean-pages-tree',
+      `Cleans your pages tree in the case the live one differs with the draft one,
```

Comment:
This should be called `sync-page-tree`. The description should be:

"Synchronizes the published page tree with the draft one, across all locales. If there is a discrepancy in the layout of the draft and published trees, the draft tree wins."

Reply:
Modified